examples/kubeflow-mlops/README.md

```diff
@@ -0,0 +1,146 @@
+# Kubeflow on EKS
+The Kubeflow project is dedicated to making deployments of machine learning (ML) workflows on Kubernetes simple, portable and scalable.
+Our goal is not to recreate other services, but to provide a straightforward way to deploy best-of-breed open-source systems for ML to diverse infrastructures.
+Anywhere you are running Kubernetes, you should be able to run Kubeflow.
+
+This example deploys the following resources
+
+- Creates EKS Cluster Control plane with public endpoint (for demo purpose only) with a managed node group
+- Deploys application load balancer and EBS SCI driver
+- Deploy Kubeflow (v1.5.1) on the EKS cluster
```

Comment:
Remove the version in the docs. We may have to remember to change the docs every-time we update the add-on version